koku/api/settings/view.py

```diff
@@ -14,9 +15,13 @@
 
 from api.common.permissions.settings_access import SettingsAccessPermission
 from api.settings.tag_management import TagManagementSettings
+from currency.settings import CurrencySettings
 
 LOG = logging.getLogger(__name__)
-SETTINGS_GENERATORS = {"tag-management": TagManagementSettings}
+if settings.DEVELOPMENT:
```

Comment:
Unleash will _not_ work here, at least, not the way in which we would want. This variable is initialized during Pod startup and then never touched again. For this logic to work with Unleash, we would need to flip a flag in Unleash, then restart the pod.

Every time the `settings` api is accessed, we need to check if things are enabled in Unleash.

The logic probably needs to live in the `_build_settings` method in this file below.